practice-1/main.cpp

```diff
@@ -14,28 +15,19 @@ class Music {
     const string artist_name;
     unsigned int year;
 
-    Music() {
-        music_id = "";
-        artist_name = "";
-        year = 0;
-    };
+    Music() : music_id(""), artist_name(""), year(0) {};
 
-    Music(string _music_id, string _artist_name, unsigned int _year) {
-        music_id = _music_id;
-        artist_name = _artist_name;
-        year = _year;
-    };
+    Music(string _music_id, string _artist_name, unsigned int _year) :
+            music_id(std::move(_music_id)),
+            artist_name(std::move(_artist_name)),
+            year(_year) {};
 
     string get_artist() {
         return artist_name;
     };
 
     bool operator==(Music &m) {
-        bool is_equal = true;
-        is_equal = is_equal && (m.music_id == music_id);
-        is_equal = is_equal && (m.artist_name == artist_name);
-        is_equal = is_equal && (m.year == year);
-        return is_equal;
+        return (m.music_id == music_id && m.artist_name == artist_name && m.year == year);
```

Comment:
More concise and reads more like English:

`return (music IDs are equal) AND (artist names are equal) AND (years are equal)`

Compare with:

```
is_equal = is_equal && (music IDs are equal)
is_equal = is_equal && (artist names are equal)
```

If you were talking like that in English, well...